src/Misc/fileTools.ts

```diff
@@ -164,7 +165,7 @@ export class FileTools {
 
         if (typeof Image === "undefined" || (engine?._features.forceBitmapOverHTMLImageElement ?? false)) {
             FileTools.LoadFile(url, (data) => {
-                engine!.createImageBitmap(new Blob([data], { type: mimeType }), { premultiplyAlpha: "none" }).then((imgBmp) => {
+                engine!.createImageBitmap(new Blob([data], { type: mimeType }), { premultiplyAlpha: "none", ...imageBitmapOptions }).then((imgBmp) => {

```

Comment:
this would crash if imageBitmapOptions is undefined.

could you replace `...imageBitmapOptions` by `...(imageBitmapOptions || {})`